Lib/test/test_tools/test_pathfix.py

```diff
@@ -14,16 +14,24 @@ class TestPathfixFunctional(unittest.TestCase):
     script = os.path.join(scriptsdir, 'pathfix.py')
 
     def setUp(self):
-        self.temp_file = support.TESTFN
-        self.addCleanup(support.unlink, support.TESTFN)
+        self.temp_directory = support.TESTFN + '.d'
+        self.addCleanup(support.rmtree, self.temp_directory)
+        os.mkdir(self.temp_directory)
+        self.temp_file = self.temp_directory + os.sep + \
+                         os.path.basename(support.TESTFN) + '-t.py'
+        self.addCleanup(support.unlink, self.temp_file)
+
+    def pathfix(self, shebang, pathfix_flags, exitcode=0, stdout='', stderr='',
+                target_file=''):
```

Comment:
I suggest to rename target_file to filename.

Reply:
Fixed in 07f7158